AsyncDisplayKitTests/ASDisplayNodeTests.m

```diff
@@ -400,6 +401,7 @@ - (void)checkValuesMatchSetValues:(ASDisplayNode *)node isLayerBacked:(BOOL)isLa
   XCTAssertEqual(YES, node.clipsToBounds, @"clipsToBounds broken %@", hasLoadedView);
   XCTAssertEqual(NO, node.opaque, @"opaque broken %@", hasLoadedView);
   XCTAssertEqual(YES, node.needsDisplayOnBoundsChange, @"needsDisplayOnBoundsChange broken %@", hasLoadedView);
+  XCTAssertEqual(NO, node.allowsGroupOpacity, @"allowsEdgeAntialiasing broken %@", hasLoadedView);
```

Comment:
@levi Thanks for adding this new bridged property, it's great to have!  Indeed these two assertions both need their string updated